tests/test_vision_classifier.py

```diff
@@ -33,6 +34,15 @@
     columns=["path", "label", "fold"],
 )
 
+numpy_image1 = Image.open(f"{TEST_PATH}/resources/images/val/cat/cat3.jpg")
+numpy_one_image = np.array(numpy_image1, "uint8")
+shape = numpy_one_image.shape
+numpy_one_image = numpy_one_image.reshape(1, shape[0], shape[1], 3)
+numpy_image1 = np.array(numpy_image1.resize((300, 300)), "uint8")
+numpy_image2 = Image.open(f"{TEST_PATH}/resources/images/val/dog/dog3.jpg")
+numpy_image2 = np.array(numpy_image2.resize((300, 300)), "uint8")
+numpy_data = np.stack([numpy_image1, numpy_image2]*8)
```

Comment:
I have a couple of questions here: first, why are there two cases for when there is one image and for when there are several? actually the case of one image is identical only that in the first dimension of the array there is one image instead of several but then the same loaders and the same functions are used, so if it works for the case of an array with 5 images there is no need to test for the case of an array with 1 image. And secondly, all these are constants that we are going to use in the tests, so the variable names should be in capital letters.

Reply:
My intention was to test whether an image with a resolution different from the model's can be loaded. However, upon further consideration, I realized that I can do that by increasing the size of the image for numpy_data